build.gradle

```diff
@@ -28,7 +28,7 @@ repositories {
 
 group = "com.electriccloud"
 description = "Plugins : EC-ESX"
-version = "2.1.5"
+version = "2.1.6"
```

Comment:
the third digit is for when we fix bugs. Lets bump up the version number to 2.2. Unless these changes are back-ward incompatible in which case to 3.0.